sparse/sfold.go

```diff
@@ -38,18 +43,16 @@ func FoldFile(childFileName, parentFileName string) error {
 	if err != nil {
 		panic("Failed to open childFile, error: " + err.Error())
 	}
-	defer childFileIo.Close()
```

Comment:
If get blocksize or get exts errors out below, files won't be closed.

Also, need another commit to change all those `panic` into `return error`. Now sfold is in the main process, they cannot just panic.